welcome/index.adoc

```diff
@@ -123,6 +123,8 @@ Use the
 xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[*Topology* view]
 to see your applications, monitor status, connect and group components, and modify your code base.
 
+- **xref:../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#understanding-service-binding-operator[Use the Service Binding Operator]**: The Service Binding Operator enables application developers to easily bind workloads together with Operator-managed backing services by automatically collecting and sharing binding data with the workloads. It enriches the development lifecycle with a consistent and declarative service binding method that eliminates discrepancies in cluster environments.
```

Comment:
Does `It` refer to the workloads, the Operator managed service, or the Service Binding Operator? 

This is how I read the sentence, feel free to apply these changes if I am correct:

`The Service Binding Operator improves the development lifecycle with a consistent and declarative service binding method that prevents discrepancies in cluster environments.` 

Note my verb change. I think `enrich` is borderline anthropomorphic and is a strong verb to use for software. Same with `eliminate.` So what do you think about my substitutes: `improves` and `prevents` ?

Reply:
'Use the Service Binding Operator' does not give the full context for a person who is simply skimming the page. Suggest adding more context. How about: Use the Service Binding Operator to connect workloads with backing services/ Simply - Connect your workloads to backing services.

Reply:
@jc-berger Lovely suggestions! Thanks again :)
I am incorporating the verb suggestion `improves`, but retaining `eliminates`.

Reply:
@Preeticp Wow, thanks for that suggestion! Love it :)